src/aspect.ts

```diff
@@ -18,11 +19,17 @@ export interface WatchfulAspectProps {
    */
   readonly dynamodb?: boolean;
 
+  /**
+   * Automatically watch AWS firehose in the scope.
+   * @default true
+   */
+  readonly firehose?: boolean;
+
   /**
    * Automatically watch AWS Lambda functions in the scope.
    * @default true
    */
-  readonly lambda?: boolean;
+  readonly lambdaFn?: boolean;
```

Comment:
Why is this changing? Some naming conflict?

Reply:
I believe I ran into issues because the variable is the same name as the import for the lambda package